jetty-start/src/main/resources/org/eclipse/jetty/start/usage.txt

```diff
@@ -60,17 +60,20 @@ Report Commands:
 
   --dry-run
                    Prints the command line that start.jar generates,
-                   then exits.
+                   in a format usable by the unix sh command, then exits.
                    This may be used to generate command lines into scripts:
                      $ java -jar start.jar --dry-run > jetty.sh
 
   --dry-run=<part>(,<part>)*
-                   Prints specific parts of the command line. The parts are:
+                   Prints specific parts of the command line in a format usable by
+                   a the unix sh command. The parts are:
                      o  "java" - the JVM to run
                      o  "opts" - the JVM options (e.g. -D, -X and -XX flags)
                      o  "path" - the JVM class-path and/or the JPMS module-path
                      o  "main" - the main class to run
                      o  "args" - the arguments passed to the main class
+                     o  "multipart" - a fake part that causes the command to be
```

Comment:
It's `multiline`! 😂

Reply:
`multiline` is not necessary.

If you are working on a shell you can accomplish this output on any already released version of Jetty now.

```
$ java -jar /path/to/jetty-home/start.jar --dry-run | xargs printf "%s\n"
```

A useful technique when troubleshooting the behavior of our quoting on various shells too (as the string to arg conversion for that shell is performed through the pipe to xargs)